RealmSwift/LinkingObjects.swift

```diff
@@ -96,6 +96,15 @@ import Realm
         return notFoundToNil(index: rlmResults.indexOfObject(with: predicate))
     }
 
+    /**
+     Returns the index of the first object matching the given query, or `nil` if no objects match.
+
+     - parameter query: The query with which to filter the objects.
```

Comment:
The parameter to this function isn't a query. In general these docs aren't really going to give anyone any clue how to call this function.